esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -2231,14 +2231,8 @@ void WsWuInfo::getWorkunitCpp(const char* cppname, const char* description, cons
     Owned<IFile> cppfile = createIFile(rfn);
     if (!cppfile)
         throw MakeStringException(ECLWATCH_CANNOT_OPEN_FILE, "Cannot open %s.", description);
```

Comment:
opening the file and error checking like this (that happens each time before readFileContent) could move into readFileContent if it just took a sourceFilename and did the createIFile itself.

Reply:
Will do